packages/cli/commands/project/cloneApp.js

```diff
@@ -0,0 +1,171 @@
+const fs = require('fs');
+const path = require('path');
+const {
+  addAccountOptions,
+  addConfigOptions,
+  getAccountId,
+  addUseEnvironmentOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { loadAndValidateOptions } = require('../../lib/validation');
+const { i18n } = require('../../lib/lang');
+const {
+  fetchPublicAppOptions,
+  selectPublicAppPrompt,
+} = require('../../lib/prompts/selectPublicAppPrompt');
+const { promptUser } = require('../../lib/prompts/promptUtils');
+const { poll } = require('../../lib/polling');
+const {
+  uiLine,
+  uiCommandReference,
+  uiAccountDescription,
+} = require('../../lib/ui');
+const SpinniesManager = require('../../lib/ui/SpinniesManager');
+const {
+  logApiErrorInstance,
+  ApiErrorContext,
+} = require('../../lib/errorHandlers/apiErrors');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
+const { isAppDeveloperAccount } = require('../../lib/accountTypes');
+const {
+  cloneApp,
+  checkCloneStatus,
+  downloadClonedProject,
+} = require('@hubspot/local-dev-lib/api/projects');
+const { getCwd, isValidPath } = require('@hubspot/local-dev-lib/path');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const { getAccountConfig } = require('@hubspot/local-dev-lib/config');
+const { extractZipArchive } = require('@hubspot/local-dev-lib/archive');
+
+const i18nKey = 'commands.project.subcommands.cloneApp';
+
+exports.command = 'clone-app';
```

Comment:
Should this be `clone-app [--appId] [--location]`?

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1085/commits/80217fc0b34dc2fdf1b979b78f2c19967105fbbe